src/lwc/geFormField/geFormField.js

```diff
@@ -304,38 +300,48 @@ export default class GeFormField extends LightningElement {
      * @param value Value to set on the field.
      */
     @api
-    setValue(value) {
-        if (this.isLookup) {
-            const lookup = this.template.querySelector('c-ge-form-field-lookup');
-            if (value) {
-                const displayValue =
-                    data[this.sourceFieldAPIName.replace('__c', '__r')].Name;
-                lookup.setSelected({value, displayValue});
+    load(data) {
+        let value;
+        if (data.hasOwnProperty(this.sourceFieldAPIName)) {
+            if (data[this.sourceFieldAPIName].hasOwnProperty('value')) {
+                value = data[this.sourceFieldAPIName].value;
             } else {
-                lookup.reset();
+                value = data[this.sourceFieldAPIName];
             }
         } else {
-            this.value = value;
+            return;
         }
-    }
 
-    /**
-     * Load a value into the form field.
-     * @param data  An sObject potentially containing a value to load.
-     */
-    @api
-    load(data) {
-        const value = data[this.sourceFieldAPIName];
-        this.setValue(value);
+        if (value === null) {
```

Comment:
@ptewson-sfdo This change actually fixes the first bug in the WI I was working on, so I'll scale back my changes to just cover the look-up issue only.